src/overlays/actors/ovl_En_Dha/z_en_dha.h

```diff
@@ -6,9 +6,28 @@
 
 struct EnDha;
 
+typedef void (*EnDhaActionFunc)(struct EnDha*, GlobalContext*);
+
 typedef struct EnDha {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x214];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[4];
+    /* 0x01A8 */ Vec3s transitionDrawTable[4];
+    /* 0x01C0 */ u8 unk_1C0;
+    /* 0x01C1 */ char unk_1C1[0x3];
+    /* 0x01C4 */ EnDhaActionFunc actionFunc;
+    /* 0x01C8 */ s16 unk_1C8;
+    /* 0x01CA */ s16 unk_1CA;
+    /* 0x01CC */ u8 unk_1CC;
+    /* 0x01CD */ char unk_1CD[1];
```

Comment:
unk_1C1 & unk_1CD are both padding that should still match when removed. They likely are not real fields and are just the gaps.